code/procs/jobprocs.dm

```diff
@@ -547,18 +547,45 @@
 
 	return
 
-/// Equip items from sensory traits
-/mob/living/carbon/human/proc/equip_sensory_items()
```

Comment:
The reason it was called `equip_sensory_items` is because it doesn't handle equipping items from any other traits like pilot or trinkets as those happen at a different position in the jobproc spaget.